web/react/components/more_direct_channels.jsx

```diff
@@ -206,7 +206,7 @@ export default class MoreDirectChannels extends React.Component {
 
         return (
             <Modal
-                className='modal-direct-channels'
+                className='more-modal'
```

Comment:
This should actually be dialogClassName when we're using React-Bootstrap Modals. I just found that out while doing all the stuff with UserSettings